src/basilisp/importer.py

```diff
@@ -412,7 +404,7 @@ def exec_module(self, module: types.ModuleType) -> None:
         else:
             try:
                 self._exec_cached_module(fullname, spec.loader_state, path_stats, ns)
-            except (EOFError, ImportError, IOError, OSError) as e:
+            except (EOFError, ImportError, OSError) as e:
```

Comment:
[TIL `IOError` is just an alias of `OSError`.](https://docs.python.org/3/library/exceptions.html#IOError)